eve/workers/pod-builder/pod.yaml

```diff
@@ -28,14 +28,17 @@ spec:
         - name: worker-workspace
           mountPath: /home/eve/workspace
     - name: platform-dind-daemon
-      image: docker:18.09.6-dind
+      image: docker:19.03.2-dind
       resources:
         requests:
           cpu: "3"
           memory: 2Gi
         limits:
           cpu: "3"
           memory: 2Gi
+      env:
+        - name: DOCKER_TLS_CERTDIR
```

Comment:
Should explain why this in the commit message (you only talk about bumping the version)

Reply:
I'm curious for this one as well :popcorn:

Reply:
done